das-jdbc/src/main/java/org/wso2/das/jdbcdriver/common/ServiceUtil.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.das.jdbcdriver.common;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.util.StringTokenizer;
+import java.util.regex.Pattern;
+
+/**
+ * This class have a collection of Utility functions required by the Driver classes.
+ */
+public class ServiceUtil {
+
+    /**
+     * Get a object of the given sql data type.
+     *
+     * @param sqlTypeName Sql Type input
+     */
+    public static Object getLiteral(String sqlTypeName) {
```

Comment:
A switch case statement would be more efficient here than the if block. Also, we could consider returning the value directly rather than temporarily assigning it to the "retval" field.